index.js

```diff
@@ -135,6 +135,14 @@ function compression(options) {
         return
       }
 
+      // Don't compress for Cache-Control: no-transform
+      // https://tools.ietf.org/html/rfc7234#section-5.2.1.6
+      var cacheControl = res.getHeader('Cache-Control') || ''
```

Comment:
Yes, there is no existing lib I know of. I was going to write one and put it in the jshttp lib and use it for the implementation of the feature request, though.